lib/puppet/pops/types/type_mismatch_describer.rb

```diff
@@ -501,23 +524,28 @@ def self.singleton
       @singleton ||= new
     end
 
+    def tense_deprecated
+      Puppet.warn_once(:deprecation, 'typemismatch#tense', "Passing a 'tense' argument to the TypeMismatchDescriber is deprecated and ignored. Everything is now reported using present tense")
+    end
+
     # Validates that all entries in the give_hash exists in the given param_struct, that their type conforms
     # with the corresponding param_struct element and that all required values are provided.
     #
     # @param subject [String] string to be prepended to the exception message
     # @param params_struct [PStructType] Struct to use for validation
     # @param given_hash [Hash<String,Object>] the parameters to validate
     # @param missing_ok [Boolean] Do not generate errors on missing parameters
-    # @param tense [Symbol] the symbol :present or :past
+    # @param tense [Symbol] deprecated and ignored
```

Comment:
wat? Are the symbols `:deprecated` and `:ignored` supported. Is that to maintain the API? Or, is this to say something else?